frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -24,26 +24,26 @@ class _CreateTripWidgetState extends State<CreateTripWidget> {
   bool _enabled = false;
 
   String place;
-  int radius = 0;
-  String newInformation = 'Grabbed info placed here.';
-  void submitTrip() {
-    setState(() {
-      tripService.createTrip(Trip(
+  Future<Trip> submitTrip() async{
+    Trip trip = await tripService.createTrip(Trip(
         name: place,
-        id: radius.toString(),
         // Everywhere Seattle now.
         placesApiPlaceId: 'ChIJVTPokywQkFQRmtVEaUZlJRA',
       ));
-      newInformation = "$place ${radius}km";
-    });
+    return trip;
   }
 
   @override
   Widget build(BuildContext context) {
     var _onPressed;
     if (_enabled) {
       _onPressed = () {
-        submitTrip();
+        submitTrip().then((trip) {
+     Navigator.pushNamed(context, Router.createTripViewRoute(trip.id));
+    }, onError: (error) {
+      print(error);
```

Comment:
Rather than print an error (to the dev console), please surface an error/warning to the user, e.g., using a [`SnackBar`](https://api.flutter.dev/flutter/material/SnackBar-class.html).